internal/db/bootstrap_logs_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2025 Canonical.
+
+package db_test
+
+import (
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/frankban/quicktest/qtsuite"
+	"github.com/google/uuid"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/testutils/jimmtest"
+)
+
+type bootstrapLogsSuite struct {
```

Comment:
We don't seem to need a separate test suite here if you want to just re-use `dbSuite`